README.md

```diff
@@ -16,18 +16,25 @@ import '@github/auto-complete-element'
 
 ```html
 <auto-complete src="/users/search" aria-owns="users-popup">
-  <input type="text" data-autocomplete-autofocus>
+  <input type="text">
   <ul id="users-popup"></ul>
 </auto-complete>
 ```
 
 The server response should include the items that matched the search query.
 
 ```html
-<li role="option" data-autocomplete-value="@hubot">Hubot</li>
-<li role="option" data-autocomplete-value="@bender">Bender</li>
-<li role="option" data-autocomplete-value="@bb-8">BB-8</li>
-<li role="option" data-autocomplete-value="@r2d2" aria-disabled="true">R2-D2 (powered down)</li>
+<li role="option">Hubot</li>
+<li role="option">Bender</li>
+<li role="option">BB-8</li>
+<li role="option" aria-disabled="true">R2-D2 (powered down)</li>
+```
+
+The `data-autocomplete-value` attribute can be used to define the value for a
+result item whose display text needs to be different:
+
+```html
+<li role="option" data-autocomplete-value="r2d2">R2-D2 (powered down)</li>
```

Comment:
Maybe we should use `bb8` in this example so it's not confused with the `aria-disabled` example above.